docs/operating-scylla/procedures/cluster-management/add-dc-to-existing-dc.rst

```diff
@@ -1,13 +1,30 @@
 Adding a New Data Center Into an Existing ScyllaDB Cluster
 ***********************************************************
 
+.. caution::
+
+    The procedure described below applies to clusters where Raft-based topology 
+    is enabled. Raft-based topology is automatically enabled in new clusters.
+
+    If you've upgraded an existing cluster from version 5.4, ensure that you 
+    :doc:`manually enabled Raft-based topology </upgrade/upgrade-opensource/upgrade-guide-from-5.4-to-6.0/enable-raft-topology>`.
```

Comment:
Can we use "Consistent Topology Updates" instead of "Raft-based topology"?
Users should not care about Raft.